test/elements/attachment-element.js

```diff
@@ -25,15 +25,15 @@ describe('attachment fields', () => {
 
     value.displayValue.should.eql('2 Attachments');
 
-    value.searchableValue.should.eql('attachment caption a different attachment caption');
+    value.searchableValue.should.eql('test_one.pdf test_two.pdf');
 
     value.toJSON().should.eql([ 
       {
          'attachment_id': '2b1fa188-39f6-4540-8bcd-9c6641e7749b',
-         'caption': 'attachment caption'
+         'name': 'test_one.pdf'
       },
       {   
-        'caption': 'a different attachment caption',
+        'name': 'test_two.pdf',
```

Comment:
I guess this answers my question.  I would prefer `filename` or `file_name` because it seems more descriptive IMHO.